website/docs/docs/quickstarts/dbt-cloud/redshift-qs.md

```diff
@@ -0,0 +1,285 @@
+
+    ```sql
+    create schema if not exists jaffle_shop;
+    create schema if not exists stripe;
+    ```
+
+8. Now create the tables in your schema with these queries using the statements below.  These will be populated as tables in the respective schemas.
+
+    ```sql
+    create table jaffle_shop.customers(
+        id integer,
+        first_name varchar(50),
+        last_name varchar(50)
+    );
+
+    create table jaffle_shop.orders(
+        id integer,
+        user_id integer,
+        order_date date,
+        status varchar(50),
+        _etl_loaded_at timestamp default current_timestamp
+    );
+
+    create table stripe.payment(
+        id integer,
+        orderid integer,
+        paymentmethod varchar(50),
+        status varchar(50),
+        amount integer,
+        created date,
+        _batched_at timestamp default current_timestamp
+    );
+    ```
+
+9. Now we need to copy the data from S3. Make sure to update the S3 location, iam role, and region. You can find the S3 and iam role in your outputs from the CloudFormation stack.
+
+    ```sql
+    copy jaffle_shop.customers( id, first_name, last_name)
+    from 's3://dbt-data-lake-xxxx/jaffle_shop_customers.csv'
+    iam_role 'arn:aws:iam::XXXXXXXXXX:role/RoleName'
+    region 'us-east-1'
+    delimiter ','
+    ignoreheader 1
+    acceptinvchars;
+       
+    copy jaffle_shop.orders(id, user_id, order_date, status)
+    from 's3://dbt-data-lake-xxxx/jaffle_shop_orders.csv'
+    iam_role 'arn:aws:iam::XXXXXXXXXX:role/RoleName'
+    region 'us-east-1'
+    delimiter ','
+    ignoreheader 1
+    acceptinvchars;
+
+    copy stripe.payment(id, orderid, paymentmethod, status, amount, created)
+    from 's3://dbt-data-lake-xxxx/stripe_payments.csv'
+    iam_role 'arn:aws:iam::XXXXXXXXXX:role/RoleName'
+    region 'us-east-1'
+    delimiter ','
+    ignoreheader 1
+    Acceptinvchars;
+    ```
+
+    Ensure that you can run a `select *` from each of the tables with the following code snippets.
+
+    ```sql 
+    select * from jaffle_shop.customers;
+    select * from jaffle_shop.orders;
+    select * from stripe.payment;
+    ```
+## Connect dbt Cloud to Redshift 
+1. Create a new project in [dbt Cloud](https://cloud.getdbt.com/). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
+2. Enter a project name and click **Continue**.
+3. For the warehouse, click **Redshift** then **Next** to set up your connection.
+4. Enter your Redshift settings. Reference your credentials you saved from the CloudFormation template.
+    - **Hostname** &mdash; Your entire hostname. Make sure to drop the `http://` at the beginning and any trailing slashes at the end.
```

Comment:
The output from Redshift doesn't include the http:// or trailing slashes, so this note may be extraneous or lead to confusion.